test/rebar_utils_SUITE.erl

```diff
@@ -118,3 +120,14 @@ special_task_do(_Config) ->
                                                                         "do",
                                                                         "bar,",
                                                                         "baz"]).
+sh_don_not_miss_messages(_Config) ->
```

Comment:
Seems like the name of the function was meant to be `sh_do_not_miss_messages` instead of `sh_don_not_miss_messages`